projects/1.18-fabric/assets/incantationem/incantationem/lang/zh_cn.json

```
@@ -0,0 +1,43 @@
+{
+  "enchantment.incantationem.bunnys_hop": "兔跳",
+  "enchantment.incantationem.swine_bane": "猪类灾祸",
+  "enchantment.incantationem.charmed": "庇佑",
+  "enchantment.incantationem.decay": "破败",
+  "enchantment.incantationem.forging_touch": "煅烧之触",
+  "enchantment.incantationem.last_stand": "背水一战",
+  "enchantment.incantationem.magnetic": "磁性",
+  "enchantment.incantationem.reaping_rod": "收割之竿",
+  "enchantment.incantationem.retreat": "撤退",
+  "enchantment.incantationem.venomous": "淬毒",
+
+  "enchantment.incantationem.recklessness": "鲁莽诅咒",
+  "enchantment.incantationem.thunder": "引雷诅咒",
+  "enchantment.incantationem.tough_luck": "不幸诅咒",
+
+
+  "message.incantationem.decay.applied": "*破败效果已施加！*",
+  "message.incantationem.forging_touch.applied": "*煅烧之触效果已施加！*",
+  "message.incantationem.last_stand.applied": "*背水一战效果已施加！*",
+  "message.incantationem.reaping_rod.applied": "*收割之竿效果已施加！*",
+  "message.incantationem.retreat.applied": "*撤退效果已施加！*",
+  "message.incantationem.venomous.applied": "*淬毒效果已施加！*",
+
+  "message.incantationem.thunder.applied": "*引雷诅咒出现了！：（*",
+
+  "enchantment.incantationem.bunnys_hop.desc": "在泥土/草方块类方块上提供跳跃提升效果。",

```

```suggestion
  "enchantment.incantationem.bunnys_hop.desc": "站在泥土/草方块类方块上提供跳跃提升效果。",
```
试试别的模组带有minecraft:dirt标签能不能生效，如果不生效就这样吧